ruby_programming/basic_ruby/project_advanced_building_blocks.md

```diff
@@ -35,6 +35,9 @@ There's also [an entry on Bubble Sort on Wikipedia](http://en.wikipedia.org/wiki
 ### Student Solutions
 Submit a link below to this [file](https://github.com/TheOdinProject/curriculum/blob/master/ruby_programming/basic_ruby/project_advanced_building_blocks.md) on the ruby course github repo with your files in it by using a pull request. See the section on [Contributing](https://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
+<details markdown="block">
+  <summary> Show Student Solutions </summary>
+
 <details markdown="block">
   <summary> Show Student Solutions </summary>
```

Comment:
Looks like this file already had the toggle. Could you remove the duplication here?

Reply:
Not sure why the preview is showing it that way. But when I go to edit it only has one `<details>` element

Reply:
Weird. I'll trust the edit view over the diff view.